### DISCUSSION TITLE
Possible to Create Network Graphs as Seen in Highchart 'networkgraph'?

### DISCUSSION THREAD

**Discussion by Myles-Harrand-02** [zauberzeug/nicegui-highcharts#8] [Q&A]
### Question

Hi, I have been playing around with NiceGUI and I wondered if anyone had gotten a [networkgraph](https://api.highcharts.com/highcharts/plotOptions.networkgraph) to work as seen in Highchart? I understand that it is a separate module so may not be available. Translating the demo code into NiceGUI code doesn't produce any results.

**Comment by falkoschindler** [comment]
Hi @Myles-Harrand-02,

Would you like to share the translated demo code?
